pkg/pokt/pokt_v0/service.go

```diff
@@ -1,9 +1,18 @@
 package pokt_v0
 
-import "os-gateway/pkg/pokt/pokt_v0/models"
+import (
+	"os-gateway/pkg/pokt/pokt_v0/models"
+	"time"
+
+	"github.com/jellydator/ttlcache/v3"
+)
 
 type PocketService interface {
 	GetSession(req *models.GetSessionRequest) (*models.GetSessionResponse, error)
 	SendRelay(req *models.SendRelayRequest) (*models.SendRelayResponse, error)
 	GetLatestBlockHeight() (*models.GetLatestBlockHeightResponse, error)
 }
+
+type CacheService interface {
```

Comment:
make a new package for this, like ttl_cache (outside of pokt)

Reply:
a ttl_cache can be used in multiple places, including other HTTP clients for example

Reply:
(also, it should be using same function signature as the generic cache). We are just creating an interface for mocks/tests